content/projects/oop/animals/part2/_index.md

```diff
@@ -115,39 +116,47 @@ Test -> Does cat eat food should Fail
 
 Use Jasmine to test your code.
 
-1. Update `Animal` super class `eats()` method to return a String, "Food".
-2. Update `Dog` class `sounds()` method to return a `String`, "Bark".
-3. Update `Cat` class `sounds()` method to return a `String`, "Meow".
-4. Make sure the `makeAllSounds()` method still works properly.
+Then create tests for your `eats()` and `sounds()` methods on the `Dog` and `Cat` classes with Jasmine.
 
 ```
-// JavaScript
 
-Dog dog = new Dog()
 
-dog.eats()    // -> 'Food'
-dog.sounds() // -> 'Bark'
+//Dog Tests
+let dog = new Dog();
+Test -> Does dog eat Food should Pass
```

Comment:
we need to change all these examples for all the languages, since from the first part of the project the name of the animal is used in the eat() method, eg:
dog.eat() // -> 'Rax eats'